### ISSUE TITLE
Install failing due to broken or incompatible module

### ISSUE THREAD

**Issue by adamtimmins** [DefectDojo/django-DefectDojo#2166]
**Bug description**
Currently attempting to install using the script setup.bash. Using the stand-alone option (3). I already have the MySQL setup and it seems to be connecting fine without any issue. I have tried to install on both Ubuntu 18 and 16 and both have failed at the same point. 

Just as it's fetching the packages it fails with trying to obtain a certain package:
```bash
error marked@0.8.2: The engine "node" is incompatible with this module. Expected version ">= 8.16.2". Got "6.17.1"
error Found incompatible module.
```

**Steps to reproduce**
1. Run the setup.bach select the standalone option.
2. Entering the database details
3. Leave everything else as default.
4. Allow the installer script to run.

**Expected behavior**
The application to successfully install.

**Deployment method** *(select with an `X`)*
- [ ] Kubernetes
- [ ] Docker
- [X] setup.bash / legacy-setup.bash

**Environment information**
 - Ubuntu 18.04 and Ubuntu 16.04

**Console logs** (optional)
```bash
warning package.json: No license field
info No lockfile found.
warning No license field
[1/5] Validating package.json...
warning No license field
[2/5] Resolving packages...
warning @yarn_components/mocha > mkdirp@0.3.5: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
warning @yarn_components/mocha > jade@0.26.3: Jade has been renamed to pug, please install the latest version of pug instead of jade
warning @yarn_components/mocha > jade > mkdirp@0.3.0: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to use Promises in 1.x.)
warning @yarn_components/mocha > glob > graceful-fs@2.0.3: please upgrade to graceful-fs 4 for compatibility with current and future versions of Node.js
warning @yarn_components/mocha > glob > minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
[3/5] Fetching packages...
error marked@0.8.2: The engine "node" is incompatible with this module. Expected version ">= 8.16.2". Got "6.17.1"
error Found incompatible module.
```

**Additional context** (optional)
I'm using AWS to set up with the environment and besides installing the necessary packages for the installer script, nothing else is being run.

**Comment by adamtimmins** [comment]
Following this documentation btw:
https://defectdojo.readthedocs.io/en/latest/running-in-production.html

**Comment by valentijnscholten** [comment]
setup.bash might be out of date, fastest is to check the Dockerfiles to see what steps are needed to get it going.

**Comment by adamtimmins** [comment]
Problem resolved, just used the latest release: 1.5.4.1 instead of cloning the git repository.

Facing a different issue altogether now, issue already highlighted in [1934](https://github.com/DefectDojo/django-DefectDojo/issues/1934) & [1931](https://github.com/DefectDojo/django-DefectDojo/issues/1931l).
